tests/pipeline/test_international_markets.py

```diff
@@ -333,6 +333,51 @@ def test_currency_convert_prices(self, name, domain, calendar_name):
 
             assert_equal(result_2d, expected_result_2d)
 
+    @parameterized.expand([
+        ('US', US_EQUITIES, 'XNYS'),
+        ('CA', CA_EQUITIES, 'XTSE'),
+        ('GB', GB_EQUITIES, 'XLON'),
+    ])
+    def test_only_currency_converted_data(self, name, domain, calendar_name):
```

Comment:
We _might_ could use a direct test for `get_rates` without any non-converted columns, in case `InMemoryFXRateReader` ever sprouts its own implementation of `get_rate_scalar`; but maybe coverage measurement is sufficient to catch that if it happens.

Reply:
> We might could use a direct test for get_rates without any non-converted columns

This doesn't really make sense. `get_rates` doesn't know anything about pipeline columns. The interface it provides is for reading a dates-by-currencies block of of fx rate data.

That said, I think a reasonable check to add is to ensure that `get_rates_scalar` is always "just" syntactic sugar for calling `get_rates` with a single date and currency and extracting the value of the 1 x 1 array. I've added that check to an existing test here: https://github.com/quantopian/zipline/pull/2613/commits/b842b87e8ed1d88604de92bb8959a06dff559d07